tests/models/embedding/language/test_embedding.py

```diff
@@ -2,15 +2,20 @@
 
 Run `pytest tests/models/embedding/language/test_embedding.py`.
 """
+import os
+
 import pytest
 import torch
 import torch.nn.functional as F
 
 MODELS = [
+    "BAAI/bge-base-en-v1.5",
     "intfloat/e5-mistral-7b-instruct",
     "BAAI/bge-multilingual-gemma2",
 ]
 
+ENCODER_MODELS = ["BAAI/bge-base-en-v1.5"]
```

Comment:
```suggestion
ENCODER_ONLY_MODELS = ["BAAI/bge-base-en-v1.5"]
```

To distinguish it from encoder-decoder models, similar to what you've done in `attention` module.